src/doc/en/developer/github.rst

```diff
@@ -0,0 +1,438 @@
+  different issues can be quite problematic, and should be avoided.
+
+- Be precise: If foo does not work on macOS but is fine on Linux,
+  mention that in the title. Use the keyword option so that
+  searches will pick up the issue.
+
+- The problem described in the issue must be solvable. For
+  example, it would be silly to open an issue whose purpose was
+  "Make Sage the best mathematical software in the world". There is
+  no metric to measure this properly and it is highly subjective.
+
+- If appropriate, provide URLs to background information or sage-devel
+  conversation relevant to the issue you are reporting.
+
+
+.. _section-github-create-issue:
+
+Opening an Issue
+================
+
+Whether it's reporting a bug or planning an enhancement, `issue
+<https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues>`_
+should be opened on our Sage repo `sagemath/sage
+<https://github.com/sagemath/sage/issues>`_ on GitHub.
+
+- Think of an apt title. People scan through the titles of issues to decide
+  which ones to look into further. So write a title that concisely describes
+  what the issue is about.
+
+- Describe the issue in detail in the issue body. What is the issue? How can we
+  solve the issue? Add links to relevant issues/PRs, and other resources.
+
+  You may use GitHub mention ``@USERNAME`` to get attention from the people
+  who would be interested in the issue or has expertise in this issue.
+
+- Add appropriate labels to the created issue:
+
+  - **Type** labels with prefix ``t:`` such as ``t: bug``, ``t: enhancement``,
+    ``t: feature``, ``t: performance``, ``t: refactoring``,
+    ``t: tests``
+
+  - **Component** labels with prefix ``c:`` such as ``c: basic arithmetic``,
+    ``c: linear algebra``, ``c: geometry``, etc.
+
+  - **Priority** labels with prefix ``p:`` such as ``p: trivial / 5``,
+    ``p: minor / 4``, ``p: major / 3``, ``p: critical / 2``, and ``p: blocker / 1``
+
+  If the issue is not expected to be solved in the near future, you may add
+  ``wishlist item`` label.
+
+
+.. _section-github-create-pr:
+
+Creating a Pull Request
+=======================
+
+If you worked on an issue, and prepared a fix for a bug or wrote code for
+enhancing Sage, then you create a PR on the Sage repo `sagemath/sage
+<https://github.com/sagemath/sage/issues>`_.
+
+In addition to what were said about opening an issue, the following applies:
+
+- The title should concisely describe what the PR does. If the PR solves an
+  issue, describe briefly what the PR solves (do not simply put the issue
+  number in the title).
+
+- Explain what the PR solves in detail in the body. If the PR solves an issue,
+  you may mention the issue here.
+
+- Add type, component, and priority labels. If this PR solves an existing
+  issue, please duplicate the labels of the issue to this PR.
+
+- **Dependencies**: Use the phrase ``Depends on``, followed by the issue or PR
```

Comment:
``- Depends on`` is better because in a list the issue title is displayed

Reply:
OK. Thanks.